examples/alignment/tracking/tracking_kf_alignment.lcsim

```diff
@@ -0,0 +1,188 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lcsim xmlns:xs="http://www.w3.org/2001/XMLSchema-instance" xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/lcsim/1.0/lcsim.xsd">
+    <!-- 
+      Steering file for running KF tracking and mille alignment
+      created:  Nov 2022
+      @author Tom Eichlersmith <eichl008@umn.edu>
+      @author PF <pbutti@slac.stanford.edu>
+    -->
+    <execute>
+      
+      <!-- Enable the following if re-processing lcio files -->
+      <driver name="PreCleanupDriver"/>      
+      
+      <driver name="RfFitter"/>
+      <driver name="EcalRunningPedestal"/> 
+      <driver name="EcalRawConverter" />
+      <driver name="EcalTimeCorrection"/> 
+      <driver name="ReconClusterer" /> 
+      <driver name="CopyCluster" /> 
+      
+      <!-- 
+        SVT reconstruction drivers 
+    
+        SensorSetup associates hits with sensors without having to re-decode them.
+        TrackerHitDriver assocites clusters with tracker modules without having to re-cluster.
+      -->
+      <driver name="SensorSetup"/>
+      <driver name="TrackerHitDriver"/> 
+
+      <!--
+        Kalman tracking
+      -->
+      <driver name="KalmanPatRecDriver"/>
+      <driver name="KalmanKinkFitDriver"/>
+      <driver name="KalmanToGBLDriver"/> 
+      
+      <!--
+        Use milli+GBL to prepare alignment data file for pede
+      -->
+      <driver name="SimpleGBLTrajAliDriverKF"/>
+      
+      <!-- it has to be before GBLOutputDriver -->
+      <driver name="MultEvtVtx" />
+      
+      <!--
+      Write out histograms
+      <driver name="GBLOutputDriverKF"/>
```

Comment:
I think we might want to use the KF track parameters in the monitoring plots. We might want to discuss a bit a plan for how to generalize this in a way so we can just configure details like this in the jobs.json file. I am often running a few different steering files with different cuts on momentum and/or phi depending on what kind of sample I am running on. We should either have multiple steering paths that all have the same output file names, or we could just make hooks here that we can feed when hps-mc makes the java call. I see a few potential directions we could achieve what we need, so maybe we should discuss a bit and decide which direction to go.